catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -436,6 +446,16 @@ public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 				.show();
 	}
 
+	protected void showMergeAlert() {
+		if (adapter.getSelectedItems().size() <= 1) {
+			ToastUtil.showError(getContext(), R.string.am_merge_error);
+		} else {
+			ToastUtil.showError(getContext(), "These two projects cannot yet be merged");
```

Comment:
no "upcoming feature" or whatsoever messages through the ui.
if the backend logic isnt implemented yet, its supposed to do nothing. theres also a feature flag and this will never reach end users.

also please move 
setShowProgressBar(true);
finishActionMode();

out of this method, they have nothing to do with showing the alert. that showMergeAlert finishes the action mode is a sideeffect. just have the switch case up there call 

showAlertDialog
setShowProgressBar(true);
finishActionMode();